netspeed/data/org.mate.panel.applet.netspeed.gschema.xml.in

```diff
@@ -16,7 +16,7 @@
       <description>If true, show speed in bits instead of bytes.</description>
     </key>
     <key name="show-all-addresses" type="b">
-      <default>false</default>
+      <default>true</default>
```

Comment:
i don't think it should be enabled by default

Reply:
I still think it should. IPv6 is the new normal. Whoever disables it or doesn't want to use it has to put extra effort.